tutorials/ndvi_hotspots.ipynb

```diff
@@ -0,0 +1,1063 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## NDVI Hotspots\n",
+    "\n",
+    "In this example, we want to illustrate how to determine hot and cold spots of NDVI (Normalized difference vegetation index) over a time series, from 2016 to 2020. These information helps us to have an overview of the change in terms of plant cover over the years. While the NDVI represents if the target being observed contains live green vegetation, its hot and cold spots help determining the recovery or disappearance of plants. In this notebook, we'll see how to:\n",
```

Comment:
Can you add a second paragraph here introducing focal analysis, with a link to https://xarray-spatial.org/user_guide/focal.html?